services/datcore-adapter/src/simcore_service_datcore_adapter/api/routes/files.py

```diff
@@ -53,3 +54,27 @@ async def delete_file(
         api_secret=x_datcore_api_secret,
         obj_id=file_id,
     )
+
+
+@router.get(
```

Comment:
Do not forget to run `make openapi.json` to update the corresponding service `openapi.json`  file

Reply:
Hi @pcrespov, are you referring to the `datcore-adapter` service? We don't have an `openapi.json` for this service

Reply:
oh! intesresting.ok